channels/xk.m3u

```diff
@@ -31,11 +31,7 @@ http://stream2.rtkit.com:1935/rtk4stream/rtk4.stream/chunklist_w599618358.m3u8
 http://rtvpuls.itprishtina.com:8005/rtvpuls-eset/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/KJLm3gi.jpg" group-title="",RTV ZiK
 http://uk4.streamingpulse.com:1935/RTVZiK/RTVZiK/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/0qVIOSH.png" group-title="",Syri Vision
-rtmp://37.26.69.49:80/live/syri
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/gfdZq6D.jpg" group-title="",TV Dielli
 http://51.15.180.159:8081/play/hls/dielli/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/C1uuCgo.png" group-title="",TV Opoja
 http://163.172.211.229:8000/live/sula/sula123/77.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/C1uuCgo.png" group-title="",TV Opoja
-http://ip.opoja.tv:1935/tvopoja/tvopoja/playlist.m3u8
```

Comment:
Link works (Tested in VLC)

Reply:
Still not working. VLC player produces:
<img width="739" alt="Screenshot 2019-10-12 at 20 12 06" src="https://user-images.githubusercontent.com/7253922/66705228-b1de7180-ed2c-11e9-8076-84fe6c73be2c.png">

Reply:
For me this links plays in VLC  and also on streamtest.in
![Screenshot_2019-10-12 Test Live Streaming Streamtest in](https://user-images.githubusercontent.com/25525836/66706596-ee15d000-ed34-11e9-8af2-34fb3457b151.png)

Reply:
https://54627d4fc5996.streamlock.net/tvopoja/tvopoja/chunklist.m3u8 - Try if this one works

Reply:
@boomski this link is working fine